lib/resolveurl/plugins/streamsb.py

```diff
@@ -22,13 +23,16 @@
 
 class StreamSBResolver(ResolveGeneric):
     name = "streamsb"
-    domains = ['streamsb.net']
-    pattern = r'(?://|\.)(streamsb\.net)/(?:embed-)?([0-9a-zA-Z]+)'
+    domains = ["sbembed.com", "sbplay.org", "streamsb.net"]
```

Comment:
Can you also add `sbvideo.net` to the list of domains. I see that you have already added it to the pattern. Thanks

Reply:
Done